kuttl-tests/kind-test.sh

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
+
+set -euxo pipefail
```

Comment:
```suggestion
set -euo pipefail
```
Do we need the versbosity?

Reply:
For consistency with the existing scripts.